developer-docs-site/docs/guides/move-guides/mint-nft-cli.md

```diff
@@ -3,76 +3,428 @@ title: "Mint an NFT with Aptos CLI"
+
+In the `init_module`, the signer is always the account uploading the contract. This gets combined with:
+
+```shell
+        token::create_collection(&resource_signer, collection, description, collection_uri, maximum_supply, mutate_setting);
+
+```
+Where the `&resource_signer` is the first parameter, defined previously as a new address that has all of the attributes of the original account plus signer capability. See:
+
+```shell
+        signer_cap: account::SignerCapability,
+```
+
+The signer capability allows the module to sign autonomously. The [resource account](../resource-accounts.md) prevents anyone from getting the private key and is entirely controlled by the contract.
+
+### Module data
+
+The `ModuleData` is then initialized and *moved* to the resource account, which has the signer capability:
+
+```shell
+        move_to(resource_account, ModuleData {
+```
+
+In the `mint_nft` function, the first step is borrowing the `ModuleData` struct:
+
+```shell
+        let module_data = borrow_global_mut<ModuleData>(@mint_nft);
+```
+
+And then use the reference to the signer capability in the  `ModuleData` struct to create the `resource_signer`:
+
+```shell
+        let resource_signer = account::create_signer_with_capability(&module_data.signer_cap);
+```
+
+In this manner, you can later use the signer capability already stored in module. When you move a module and its structs into an account, they become visible in [Aptos Explorer](https://explorer.aptoslabs.com/) associated with the account.
+
+## 1. Create a collection and token
+
+Now that you have an understanding of the Aptos minting process and are starting to write smart contracts with Move, you are ready to create your first testnet NFT with the Aptos CLI.
 
-## Mint with the Aptos CLI
+In this section, we create a collection and token. This work maps to the demonstration in [create_nft.move](https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/move-examples/mint_nft/1-Create-NFT/sources/create_nft.move).
 
-Now that you are starting to write smart contracts with Move, let's create our first NFT with the Aptos CLI.
+### Create an account
 
 1. [Install the Aptos CLI](../../cli-tools/aptos-cli-tool/install-aptos-cli.md) and note its [many uses](../../cli-tools/aptos-cli-tool/use-aptos-cli.md) for later if you haven't experienced its goodness already.
 
-1. Create an account on Aptos testnet by running the following command and selecting `testnet`:
+2. Create a default (typical) account on Aptos testnet to receive the NFT by running the following command and selecting `testnet`:
+  ```shell
+  aptos init
+  ```
+
+3. Receive the output:
+  ```shell
+  Configuring for profile default
+  ```
+
+3. When prompted for a network:
+  ```shell
+  Choose network from [devnet, testnet, mainnet, local, custom | defaults to devnet]
+  ```
+  Select `testnet` by entering it and hitting return.
```

Comment:
I think most of the time they should choose devnet (testnet APTs are harder to get)

Reply:
OK, I am replacing throughout except in this output:

```
  No key given, generating key...
  Account a233bf7be2b93f1e532f8ea88c49e0c70a873d082890b6d9685f89b5e40d50c2 does not exist, you will need to create and fund the account through a community faucet e.g. https://aptoslabs.com/testnet-faucet, 
```

I don't see an analogous: https://aptoslabs.com/devnet-faucet